data/provider/src/main/java/tech/pegasys/teku/api/ValidatorDataProvider.java

```diff
@@ -62,6 +63,10 @@
   private CombinedChainDataClient combinedChainDataClient;
   private final BlockImporter blockImporter;
 
+  public static final int INTERNAL_ERROR_HTTP_CODE = 500;
+  public static final int ACCEPTED_HTTP_CODE = 202;
+  public static final int OK_HTTP_CODE = 200;
```

Comment:
Do these need to be public?  And can we use a predefined constant like from `HttpServletResponse` or one of the various `HttpStatus` classes?

Reply:
Done.